symengine/symengine_exception.h

```diff
@@ -0,0 +1,34 @@
+#ifndef SYMENGINE_EXCEPTION_H
+#define SYMENGINE_EXCEPTION_H
+
+enum symengine_exceptions_t {
+    NO_EXCEPTION,
```

Comment:
Can you explicitly set this to 0? So that the usual way of writing the function inside the if condition works everytime.